### ISSUE TITLE
NameError: name 'basestring' is not defined

### ISSUE THREAD

**Issue by vschulz** [brentp/poverlap#10]
I get error 
NameError: name 'basestring' is not defined
using python 3.10.

**Comment by brentp** [comment]
Yeah, this package is probably old enough that it won't work on python3 without some updates.
